.mergify.yml

```diff
@@ -53,6 +54,9 @@ queue_rules:
               - "status-success=ci/centos/mini-e2e/k8s-1.21"
               - "status-success=ci/centos/mini-e2e/k8s-1.22"
               - "status-success=ci/centos/mini-e2e/k8s-1.23"
+              - "status-success=ci/centos/k8s-e2e-external-storage/1.21"
+              - "status-success=ci/centos/k8s-e2e-external-storage/1.22"
+              - "status-success=ci/centos/k8s-e2e-external-storage/1.23"
```

Comment:
are we running these tests in release-3.4 branch?

Reply:
@Madhu-1 done. missed to push last time.